### ISSUE TITLE
After successfully deploying to App Engine, browser loads .appspot.com (Can't reach this page)

### ISSUE THREAD

**Issue by SurferJeffAtGoogle** [GoogleCloudPlatform/google-cloud-visualstudio#854]
```
Publishing WebApplication4 to Compute Engine.
dotnet publish -o "C:\Users\Jeffrey Rennie\AppData\Local\Temp\mabapqok.xha" -c Release

Welcome to .NET Core!
---------------------
Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.

Telemetry
--------------
The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
Microsoft (R) Build Engine version 15.3.409.57025 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  WebApplication4 -> c:\users\jeffrey rennie\source\repos\WebApplication4\WebApplication4\bin\Release\netcoreapp2.0\WebApplication4.dll
  WebApplication4 -> C:\Users\Jeffrey Rennie\AppData\Local\Temp\mabapqok.xha\
Services to deploy:

descriptor:      [C:\Users\Jeffrey Rennie\AppData\Local\Temp\mabapqok.xha\app.yaml]
source:          [C:\Users\Jeffrey Rennie\AppData\Local\Temp\mabapqok.xha]
target project:  [surferjeff-test2]
target service:  [default]
target version:  [20171129t163724]
target url:      [https://surferjeff-test2.appspot.com]


Beginning deployment of service [default]...
Building and pushing image for service [default]
Started cloud build [799af90b-1296-4dee-ab68-9e02ae809e06].
To see logs in the Cloud Console: https://console.cloud.google.com/gcr/builds/799af90b-1296-4dee-ab68-9e02ae809e06?project=surferjeff-test2
----------------------------- REMOTE BUILD OUTPUT ------------------------------
starting build "799af90b-1296-4dee-ab68-9e02ae809e06"

FETCHSOURCE
Fetching storage object: gs://staging.surferjeff-test2.appspot.com/us.gcr.io/surferjeff-test2/appengine/default.20171129t163724:latest#1512002293388982
Copying gs://staging.surferjeff-test2.appspot.com/us.gcr.io/surferjeff-test2/appengine/default.20171129t163724:latest#1512002293388982...
/ [0 files][    0.0 B/575.6 KiB]                                                
/ [1 files][575.6 KiB/575.6 KiB]                                                

Operation completed over 1 objects/575.6 KiB.                                    
BUILD
Step #0: Pulling image: gcr.io/gcp-runtimes/aspnetcorebuild@sha256:d7b7975acb374fc3a9655a4e529993e6270cfa78023885684626528bc379f8eb
Step #0: sha256:d7b7975acb374fc3a9655a4e529993e6270cfa78023885684626528bc379f8eb: Pulling from gcp-runtimes/aspnetcorebuild
Step #0: da859c16439c: Already exists
Step #0: f341f81212ab: Pulling fs layer
Step #0: 581119a9a107: Pulling fs layer
Step #0: 564d4d06dac6: Pulling fs layer
Step #0: be3958556b95: Pulling fs layer
Step #0: be3958556b95: Waiting
Step #0: 564d4d06dac6: Verifying Checksum
Step #0: 564d4d06dac6: Download complete
Step #0: 581119a9a107: Verifying Checksum
Step #0: 581119a9a107: Download complete
Step #0: be3958556b95: Verifying Checksum
Step #0: be3958556b95: Download complete
Step #0: f341f81212ab: Verifying Checksum
Step #0: f341f81212ab: Download complete
Step #0: f341f81212ab: Pull complete
Step #0: 581119a9a107: Pull complete
Step #0: 564d4d06dac6: Pull complete
Step #0: be3958556b95: Pull complete
Step #0: Digest: sha256:d7b7975acb374fc3a9655a4e529993e6270cfa78023885684626528bc379f8eb
Step #0: Status: Downloaded newer image for gcr.io/gcp-runtimes/aspnetcorebuild@sha256:d7b7975acb374fc3a9655a4e529993e6270cfa78023885684626528bc379f8eb
Starting Step #0
Finished Step #0
Step #1: Pulling image: gcr.io/cloud-builders/docker@sha256:aa949d45f505028b03147d748e88cb72ad360dd13baacc492052fd6b0239b5ec
Step #1: sha256:aa949d45f505028b03147d748e88cb72ad360dd13baacc492052fd6b0239b5ec: Pulling from cloud-builders/docker
Step #1: ae79f2514705: Pulling fs layer
Step #1: 5ad56d5fc149: Pulling fs layer
Step #1: 170e558760e8: Pulling fs layer
Step #1: 395460e233f5: Pulling fs layer
Step #1: 6f01dc62e444: Pulling fs layer
Step #1: ce78d9a4a7d7: Pulling fs layer
Step #1: a098624c8a71: Pulling fs layer
Step #1: 395460e233f5: Waiting
Step #1: 6f01dc62e444: Waiting
Step #1: ce78d9a4a7d7: Waiting
Step #1: a098624c8a71: Waiting
Step #1: 170e558760e8: Verifying Checksum
Step #1: 170e558760e8: Download complete
Step #1: 5ad56d5fc149: Verifying Checksum
Step #1: 5ad56d5fc149: Download complete
Step #1: 6f01dc62e444: Verifying Checksum
Step #1: 6f01dc62e444: Download complete
Step #1: 395460e233f5: Verifying Checksum
Step #1: 395460e233f5: Download complete
Step #1: ae79f2514705: Verifying Checksum
Step #1: ae79f2514705: Download complete
Step #1: ae79f2514705: Pull complete
Step #1: 5ad56d5fc149: Pull complete
Step #1: 170e558760e8: Pull complete
Step #1: 395460e233f5: Pull complete
Step #1: a098624c8a71: Verifying Checksum
Step #1: a098624c8a71: Download complete
Step #1: 6f01dc62e444: Pull complete
Step #1: ce78d9a4a7d7: Verifying Checksum
Step #1: ce78d9a4a7d7: Download complete
Step #1: ce78d9a4a7d7: Pull complete
Step #1: a098624c8a71: Pull complete
Step #1: Digest: sha256:aa949d45f505028b03147d748e88cb72ad360dd13baacc492052fd6b0239b5ec
Step #1: Status: Downloaded newer image for gcr.io/cloud-builders/docker@sha256:aa949d45f505028b03147d748e88cb72ad360dd13baacc492052fd6b0239b5ec
Starting Step #1
Step #1: Sending build context to Docker daemon   2.51MB


Step #1: Step 1/4 : FROM gcr.io/google-appengine/aspnetcore@sha256:365627b6182da57b249b4cceb5ecb0c22db7df2c0bd43f46d303115adf47a778
Step #1: sha256:365627b6182da57b249b4cceb5ecb0c22db7df2c0bd43f46d303115adf47a778: Pulling from google-appengine/aspnetcore
Step #1: da859c16439c: Already exists
Step #1: 41b3caecd084: Pulling fs layer
Step #1: f12f8a16d6a6: Pulling fs layer
Step #1: 41b3caecd084: Verifying Checksum
Step #1: 41b3caecd084: Download complete
Step #1: 41b3caecd084: Pull complete
Step #1: f12f8a16d6a6: Verifying Checksum
Step #1: f12f8a16d6a6: Download complete
Step #1: f12f8a16d6a6: Pull complete
Step #1: Digest: sha256:365627b6182da57b249b4cceb5ecb0c22db7df2c0bd43f46d303115adf47a778
Step #1: Status: Downloaded newer image for gcr.io/google-appengine/aspnetcore@sha256:365627b6182da57b249b4cceb5ecb0c22db7df2c0bd43f46d303115adf47a778
Step #1:  ---> 4dfd878cbc27
Step #1: Step 2/4 : ADD ./ /app
Step #1:  ---> f6307cd0d206
Step #1: Removing intermediate container 55c6332591b0
Step #1: Step 3/4 : WORKDIR /app
Step #1:  ---> 2ff82f669b4d
Step #1: Removing intermediate container 42951deee870
Step #1: Step 4/4 : ENTRYPOINT dotnet WebApplication4.dll
Step #1:  ---> Running in 08fe92a27480
Step #1:  ---> 1c9135ea052f
Step #1: Removing intermediate container 08fe92a27480
Step #1: Successfully built 1c9135ea052f
Step #1: Successfully tagged us.gcr.io/surferjeff-test2/appengine/default.20171129t163724:latest
Finished Step #1
PUSH
Pushing us.gcr.io/surferjeff-test2/appengine/default.20171129t163724:latest
The push refers to a repository [us.gcr.io/surferjeff-test2/appengine/default.20171129t163724]
7ccda8a5b406: Preparing
dff38f28221e: Preparing
aab344da8038: Preparing
5a2c5095cca1: Preparing
5a2c5095cca1: Layer already exists
aab344da8038: Layer already exists
dff38f28221e: Layer already exists
7ccda8a5b406: Pushed
latest: digest: sha256:97b617d849f356645e7222f8f146f5fc2410f75e4b48970c2c476aef83fe48b3 size: 1164
DONE
--------------------------------------------------------------------------------

Updating service [default] (this may take several minutes)...
......................................................................................................................................................................................................................................................................................................................................................................................done.
Updating service [default]...
Waiting for operation [apps/surferjeff-test2/operations/b13226e8-eb03-4274-868e-e2c93b642421] to complete...
..............done.
Stopping version [surferjeff-test2/default/20171128t170605].
Sent request to stop version [surferjeff-test2/default/20171128t170605]. This operation may take some time to complete. If you would like to verify that it succeeded, run:
  $ gcloud app versions describe -s default 20171128t170605
until it shows that the version has stopped.
done.
Deployed service [default] to [https://surferjeff-test2.appspot.com]

You can stream logs from the command line by running:
  $ gcloud app logs tail -s default

To view your application in the web browser run:
  $ gcloud app browse
Project WebApplication4 deployed to App Engine Flex.
App deployed to https://.appspot.com
```

```
Microsoft Visual Studio Community 2017 
Version 15.4.4
VisualStudio.15.Release/15.4.4+27004.2009
Microsoft .NET Framework
Version 4.7.02053

Installed Version: Community

Visual Basic 2017   00369-60000-00001-AA909
Microsoft Visual Basic 2017

Visual C# 2017   00369-60000-00001-AA909
Microsoft Visual C# 2017

Application Insights Tools for Visual Studio Package   8.9.00809.2
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2017   15.0.30925.0
ASP.NET and Web Tools 2017

ASP.NET Core Razor Language Services   1.0
Provides languages services for ASP.NET Core Razor.

ASP.NET Web Frameworks and Tools 2017   5.2.50921.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v3.0.0   15.0.30915.0
Azure App Service Tools v3.0.0

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

DeployToGae Extension   1.0
DeployToGae Visual Studio Extension Detailed Info

JavaScript Language Service   2.0
JavaScript Language Service

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 2017 - v2.9.50719.1

Microsoft Continuous Delivery Tools for Visual Studio   0.3
Simplifying the configuration of continuous build integration and continuous build delivery from within the Visual Studio IDE.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

NuGet Package Manager   4.4.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

SQL Server Data Tools   15.1.61707.200
Microsoft SQL Server Data Tools

TypeScript   2.3.5.0
TypeScript tools for Visual Studio

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio
```

Extension version 1.2.10.0.

**Comment by amanda-tarafa** [comment]
99% certain this was fixed on #820 since this error was probably due to the mismatch that existed between the Publish Dialog current project and the Select Project Dialog current project.
Not closing until @ILMTitan confirms since he's the one who worked on #820 .
@ILMTitan adding you to assignees,  PTAL.

**Comment by ILMTitan** [comment]
I believe that #820 does indeed hide this error. The problem was, previously, you could attempt to publish with no GCP project selected in the VS extension, and it would attempt to publish anyway, causing gcloud to use its own configuration. The extension would then navigate to the VS extension configured project (aka the empty string). #820 does prevent you from publishing without a selected VS extension project.
